discord.h

```diff
@@ -470,6 +475,15 @@ void discord_set_on_thread_update(struct discord *client,
  */
 void discord_set_on_thread_delete(struct discord *client,
                                   discord_on_channel callback);
+
+void discord_set_on_guild_create(struct discord *client,
```

Comment:
Could you add some brief descriptions here? Just like the functions surrounding it!